cli/src/commands/migrations/create.rs

```diff
@@ -0,0 +1,15 @@
+use crate::migrations::Migrations;
+use eyre::Context;
+
+pub async fn create(path: Option<&str>, name: &str) -> eyre::Result<()> {
+    let current_dir = std::env::current_dir().wrap_err("Failed to get current dir")?;
+
+    Migrations::new(current_dir.join(path.unwrap_or("migrations")).as_path())
```

Comment:
@flyin can we make sure that even if they call the command in a nested directory within `src/` the root is still taken as the crate's root?

Reply:
@gvidon I thought about it! I'll pass `Project` here and will use `project.path` instead of current_dir

Reply:
Done

pass Project to the command and use `project.path` as crate root